### ISSUE TITLE
Visualizing results

### ISSUE THREAD

**Issue by underbais** [morrislab/phylowgs#74]
I am trying to use your package PhyloWGS and I am stuck with step 5. View results.
 
I am running it in the cluster so there is no way I can initiate a browser like you specify at Github
https://github.com/morrislab/phylowgs
 
So, is there other ways to visualize results using your package to get the trees?

Thanks

**Comment by jwintersinger** [comment]
You have several options:

1. Run `write_results.py` and transfer the resulting files to your local machine, then run Witness from there.
2. Use something like SSHFS (https://github.com/libfuse/sshfs) to mount the cluster's filesystem locally, and again run Witness from your local machine.
3. Set up an SSH tunnel (e.g., `ssh -L 8000:localhost:8000 user@example.org`). This way, you can run a web server on the cluster (via `python2 -m SimpleHTTPServer`), but still connect to it from your local machine.

Please let me know if you need any other help!

**Comment by vivekruhela** [comment]
Hi,
I am trying to visualize results using the command:
``
but I am getting empty dashboard. All the files are generated from example data and saved in data sub-folder of witness folder. I run that command twice or refresh it but still getting empty dashboard.
Here are the logs and command:

`vivekr@sarabhai:~/clone_evolution/phylowgs/witness$ python2 -m SimpleHTTPServer`
`Serving HTTP on 0.0.0.0 port 8000 ...`
`127.0.0.1 - - [06/Jul/2019 10:06:05] "GET / HTTP/1.1" 200 -`
`127.0.0.1 - - [06/Jul/2019 10:06:05] "GET /css/dashboard.css HTTP/1.1" 200 -`
`127.0.0.1 - - [06/Jul/2019 10:06:05] "GET /css/tree.css HTTP/1.1" 200 -`
`127.0.0.1 - - [06/Jul/2019 10:06:05] "GET /js/stupidtable.min.js HTTP/1.1" 200 -`
`127.0.0.1 - - [06/Jul/2019 10:06:05] "GET /js/state_manager.js HTTP/1.1" 200 -`
`127.0.0.1 - - [06/Jul/2019 10:06:05] "GET /js/cluster_plotter.js HTTP/1.1" 200 -`
`127.0.0.1 - - [06/Jul/2019 10:06:05] "GET /js/interface.js HTTP/1.1" 200 -`
`127.0.0.1 - - [06/Jul/2019 10:06:05] "GET /js/tree_plotter.js HTTP/1.1" 200 -`
`127.0.0.1 - - [06/Jul/2019 10:06:05] "GET /js/tree_viewer.js HTTP/1.1" 200 -`
`127.0.0.1 - - [06/Jul/2019 10:06:05] "GET /js/tree_summarizer.js HTTP/1.1" 200 -`
`127.0.0.1 - - [06/Jul/2019 10:06:05] "GET /js/tree_util.js HTTP/1.1" 200 -`
`127.0.0.1 - - [06/Jul/2019 10:06:05] "GET /js/pop_vaf_plotter.js HTTP/1.1" 200 -`
`127.0.0.1 - - [06/Jul/2019 10:06:05] "GET /js/util.js HTTP/1.1" 200 -`
`127.0.0.1 - - [06/Jul/2019 10:06:05] "GET /js/witness.js HTTP/1.1" 200 -`
`127.0.0.1 - - [06/Jul/2019 10:06:07] code 404, message File not found`
`127.0.0.1 - - [06/Jul/2019 10:06:07] "GET /favicon.ico HTTP/1.1" 404 -`
`127.0.0.1 - - [06/Jul/2019 10:06:37] "GET /data/index.json HTTP/1.1" 200 -`
`127.0.0.1 - - [06/Jul/2019 10:07:29] "GET / HTTP/1.1" 200 -`
`127.0.0.1 - - [06/Jul/2019 10:07:29] "GET /js/state_manager.js HTTP/1.1" 200 -`
`127.0.0.1 - - [06/Jul/2019 10:07:29] "GET /js/stupidtable.min.js HTTP/1.1" 200 -`
`127.0.0.1 - - [06/Jul/2019 10:07:29] "GET /js/tree_viewer.js HTTP/1.1" 200 -`
`127.0.0.1 - - [06/Jul/2019 10:07:29] "GET /js/tree_plotter.js HTTP/1.1" 200 -`
`127.0.0.1 - - [06/Jul/2019 10:07:29] "GET /js/interface.js HTTP/1.1" 200 -`
`127.0.0.1 - - [06/Jul/2019 10:07:29] "GET /js/cluster_plotter.js HTTP/1.1" 200 -`
`127.0.0.1 - - [06/Jul/2019 10:07:29] "GET /js/tree_summarizer.js HTTP/1.1" 200 -`
`127.0.0.1 - - [06/Jul/2019 10:07:29] "GET /css/dashboard.css HTTP/1.1" 200 -`
`127.0.0.1 - - [06/Jul/2019 10:07:29] "GET /css/tree.css HTTP/1.1" 200 -`
`127.0.0.1 - - [06/Jul/2019 10:07:29] "GET /js/tree_util.js HTTP/1.1" 200 -`
`127.0.0.1 - - [06/Jul/2019 10:07:29] "GET /js/pop_vaf_plotter.js HTTP/1.1" 200 -`
`127.0.0.1 - - [06/Jul/2019 10:07:29] "GET /js/util.js HTTP/1.1" 200 -`
`127.0.0.1 - - [06/Jul/2019 10:07:29] "GET /js/witness.js HTTP/1.1" 200 -`
`127.0.0.1 - - [06/Jul/2019 10:07:30] "GET /data/index.json HTTP/1.1" 200 -`
`127.0.0.1 - - [06/Jul/2019 10:10:22] "GET / HTTP/1.1" 200 -`
`127.0.0.1 - - [06/Jul/2019 10:10:22] "GET /css/dashboard.css HTTP/1.1" 200 -`
`127.0.0.1 - - [06/Jul/2019 10:10:22] "GET /css/tree.css HTTP/1.1" 200 -`
`127.0.0.1 - - [06/Jul/2019 10:10:22] "GET /js/state_manager.js HTTP/1.1" 200 -`
`127.0.0.1 - - [06/Jul/2019 10:10:22] "GET /js/stupidtable.min.js HTTP/1.1" 200 -`
`127.0.0.1 - - [06/Jul/2019 10:10:22] "GET /js/cluster_plotter.js HTTP/1.1" 200 -`
`127.0.0.1 - - [06/Jul/2019 10:10:22] "GET /js/interface.js HTTP/1.1" 200 -`
`127.0.0.1 - - [06/Jul/2019 10:10:22] "GET /js/tree_plotter.js HTTP/1.1" 200 -`
`127.0.0.1 - - [06/Jul/2019 10:10:22] "GET /js/tree_viewer.js HTTP/1.1" 200 -`
`127.0.0.1 - - [06/Jul/2019 10:10:22] "GET /js/tree_summarizer.js HTTP/1.1" 200 -`
`127.0.0.1 - - [06/Jul/2019 10:10:22] "GET /js/witness.js HTTP/1.1" 200 -`
`127.0.0.1 - - [06/Jul/2019 10:10:22] "GET /js/tree_util.js HTTP/1.1" 200 -`
`127.0.0.1 - - [06/Jul/2019 10:10:22] "GET /js/pop_vaf_plotter.js HTTP/1.1" 200 -`
`127.0.0.1 - - [06/Jul/2019 10:10:22] "GET /js/util.js HTTP/1.1" 200 -`
`127.0.0.1 - - [06/Jul/2019 10:10:23] code 404, message File not found`
`127.0.0.1 - - [06/Jul/2019 10:10:23] "GET /favicon.ico HTTP/1.1" 404 -`
`127.0.0.1 - - [06/Jul/2019 10:10:25] "GET /data/index.json HTTP/1.1" 200 -`

**Comment by vivekruhela** [comment]
An update I would like to add here. I have got some visualizations but the script is hanging badly and results are not static (i.e. changing). There are multiple plots for the same x and y label which is confusing also. I have attached the figure below:
![Screenshot_2019-07-09 PhyloWGS Witness(3)](https://user-images.githubusercontent.com/13174913/60861681-d95b0500-a238-11e9-9424-189650d24b4e.png)
![Screenshot_2019-07-09 PhyloWGS Witness(4)](https://user-images.githubusercontent.com/13174913/60861682-d95b0500-a238-11e9-9c5b-efda4921f417.png)
![Screenshot_2019-07-09 PhyloWGS Witness(5)](https://user-images.githubusercontent.com/13174913/60861684-d9f39b80-a238-11e9-9910-7fbced0ae3d7.png)
![Screenshot_2019-07-09 PhyloWGS Witness(6)](https://user-images.githubusercontent.com/13174913/60861685-d9f39b80-a238-11e9-8d2f-b9a132f4ea1a.png)
